src/main/drivers/bus_spi_hal.c

```diff
@@ -260,27 +261,21 @@ bool spiIsBusBusy(SPI_TypeDef *instance)
         return false;
 }
 
-bool spiTransfer(SPI_TypeDef *instance, uint8_t *out, const uint8_t *in, int len)
+bool spiTransfer(SPI_TypeDef *instance, const uint8_t *in, uint8_t *out, int len)
```

Comment:
I'm glad someone else finds them confusing. I'll change to `rxData/txData` throughout.